mythtv/libs/libmythbase/filesysteminfo.cpp

```diff
@@ -95,6 +95,49 @@ bool FileSystemInfo::FromStringList(QStringList::const_iterator &it,
     return true;
 }
 
+bool FileSystemInfo::refresh()
```

Comment:
Pretty sure the entire of this can be done with Qt's [QStorageInfo](https://doc.qt.io/qt-5/qstorageinfo.html)

Reply:
It now uses QStorageInfo.